performance/admin.py

```diff
@@ -0,0 +1,447 @@
+                q2_deviation_reason=indicator_data['ReasonforDeviation_Q2'],
+                q2_corrective_action=indicator_data['CorrectiveAction_Q2'],
+                q2_national_comments=indicator_data['National_Q2'],
+                q2_otp_comments=indicator_data['OTP_Q2'],
+                q2_dpme_coordinator_comments=indicator_data['OTP_Q2'],
+                q2_treasury_comments=indicator_data['National_Q2'],
+
+                q3_target=indicator_data['Target_Q3'],
+                q3_actual_output=indicator_data['ActualOutput_Q3'],
+                q3_deviation_reason=indicator_data['ReasonforDeviation_Q3'],
+                q3_corrective_action=indicator_data['CorrectiveAction_Q3'],
+                q3_national_comments=indicator_data['National_Q3'],
+                q3_otp_comments=indicator_data['OTP_Q3'],
+                q3_dpme_coordinator_comments=indicator_data['OTP_Q3'],
+                q3_treasury_comments=indicator_data['National_Q3'],
+
+                q4_target=indicator_data['Target_Q4'],
+                q4_actual_output=indicator_data['ActualOutput_Q4'],
+                q4_deviation_reason=indicator_data['ReasonforDeviation_Q4'],
+                q4_corrective_action=indicator_data['CorrectiveAction_Q4'],
+                q4_national_comments=indicator_data['National_Q4'],
+                q4_otp_comments=indicator_data['OTP_Q4'],
+                q4_dpme_coordinator_comments=indicator_data['OTP_Q4'],
+                q4_treasury_comments=indicator_data['National_Q4'],
+
+                annual_target=indicator_data['AnnualTarget_Summary2'],
+                annual_aggregate_output='',
+                annual_pre_audit_output=indicator_data['PrelimaryAudited_Summary2'],
+                annual_deviation_reason=indicator_data['ReasonforDeviation_Summary'],
+                annual_corrective_action=indicator_data['CorrectiveAction_Summary'],
+                annual_otp_comments=indicator_data['OTP_Summary'],
+                annual_national_comments=indicator_data['National_Summary'],
+                annual_dpme_coordincator_comments=indicator_data['OTP_Summary'],
+                annual_treasury_comments=indicator_data['National_Summary'],
+                annual_audited_output=indicator_data['ValidatedAudited_Summary2'],
+
+                sector=indicator_data['Sector'],
+                programme_name=indicator_data['SubProgramme'],
+                subprogramme_name=indicator_data['Location'],
+                frequency=[i[0] for i in models.FREQUENCIES if i[1] == frequency][0],
+                type=indicator_data['Type'],
+                subtype=indicator_data['SubType'],
+                mtsf_outcome=indicator_data['Outcome'],
+                cluster=indicator_data['Cluster'],
+                uid=indicator_data['UID'],
+            )
+            num_imported = num_imported + 1
+        elif department_name not in not_matching_departments:
+            not_matching_departments.append(department_name)
```

Comment:
if it was a set, this conditional would be unnecessary

Reply:
done